CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 3.0.1
+
+- fix wmic removed on Windows 11 and add gwmi support
+
```

Comment:
This is awesome, I'll suggest however to target a 4.x as its a new feature and we follow semver. I'll merge some pending patches but I'd love to see this being added, many thanks for the work! 

P.S.: I had a windows environment at the time of coding the first version but now I don't anymore so its hard for me to test this.